roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -51,6 +51,16 @@
   failed_when: false
   when: openshift.common.is_containerized | bool
 
+- name: Stop rpm based services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+  - "{{ openshift.common.service_type }}-node"
+  - openvswitch
+  failed_when: false
+  when: not openshift.common.is_containerized | bool
```

Comment:
I was going to just remove the conditional on the previous task, but that would've meant that master services got restarted on rpm based installs but that's not necessary and they wouldn't be restarted. So just add a non containerized stop here.